package.json

```diff
@@ -102,6 +102,9 @@
     "react": "^7.0.1",
     "react-dom": "^17.0.1"
   },
+  "resolutions": {
```

Comment:
is this going to force every version of babel-loader to use 8.1.0? or what does it do exactly?

Reply:
Right, we could set it to  "^8.1.0" if you prefer. Or should we use the `SKIP_PREFLIGHT_CHECK=true` env variable when there are version conflicts in the packages that use our dependencies?

Reply:
let's use this fix for now as seems to be more elegant than the `SKIP_PREFLIGHT_CHECK` for now.
I've read we should change a bit how we manage the dependencies in the Monorepo and each package.json responsibility but this will be addressed later on. Thanks for the clarification!